src/test/java/org/abstractj/kalium/fixture/TestVectors.java

```diff
@@ -62,6 +62,17 @@
     public static final String Blake2_PERSONAL = "fedcba9876543210";
     public static final String Blake2_DIGEST_WITH_SALT_PERSONAL = "9479874d504f0447d43d72a969c989c34032172276ac50077e0027277c3c8d867bee9ee314c8506e4e4a9b3030b989d3eb7b4c1826c8e0d56e6aa71b1d4cf388";
 
```

Comment:
I assume that you're referring to the following test vectors:

```java
    public static final String PWHASH_MESSAGE = "Correct Horse Battery Staple";
    public static final String PWHASH_SALT = "[<~A 32-bytes salt for scrypt~>]";
```

Instead of the `Blake2_PERSONAL` & `Blake2_DIGEST_WITH_SALT_PERSONAL`